proto/warrior.proto

```diff
@@ -113,7 +113,6 @@ enum WarriorMinorGlyph {
 	GlyphOfEnduringVictory = 43400;
 	GlyphOfFuriousSundering = 45793;
 	GlyphOfIntimidatingShout = 45794;
-	GlyphOfShatteringThrow = 206953;
```

Comment:
Add this glyph id to the database overrides to remove it as selectable from the ui

Reply:
Changes Made.  I revert the glyph for now and now calculate the damage pct and extra damage for MS and Slam